.travis.yml

```diff
@@ -20,4 +20,4 @@ script:
   - npm run build_cover
 
 after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+  - cat ./coverage/coverage-remapped.lcov  | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Yes, I'm looking those part now. On my forked branch this works, failed here, continuing investigation.